gendst/data/positions.go

```diff
@@ -85,32 +85,32 @@ var Q = map[string]string{
 }
 
 // ArrayType
-type R /*Start*/ [ /*Lbrack*/ 1] /*Len*/ int /*End*/
+type R[P any] /*Start*/ [ /*Lbrack*/ 1] /*Len*/ P /*End*/
```

Comment:
Are all the type parameters needed on these examples? I think it's better to leave them as simple as possible...

Reply:
Absolutely understood! I was just trying to make them complete -- I didn't know which one you preferred!